### ISSUE TITLE
Drawer dialog doesn't have aria role (WCAG 4.1.2) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#194]
The dialog element does not have a name available to API.

ARIA dialogs and alertdialogs must have accessible names.
A label associated with a form element provides information about that element to screen readers when it receives a focus.
Ensure that each element with role="dialog" or role="alertdialog" has one of the following characteristics:
   • Non-empty aria-label attribute OR
   • aria-labelledby pointing to the element with text which is discernible to screen reader users

Please provide a descriptive accessible name (e.g. using an aria-label), so that the purpose of the dialog is clear.

![image](https://user-images.githubusercontent.com/93537804/229165014-a52d9af9-7b70-4a11-be5e-f476e733a507.png)

**Sufficient Techniques:**
- ARIA dialog and alertdialog must have an accessible name ->
https://dequeuniversity.com/rules/axe/4.3/aria-dialog-name
- Using aria-label to provide an invisible label where a visible label cannot be used ->
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14
- Using aria-labelledby to provide a name for user interface controls ->
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA16

**Success Criteria:**
- Understanding Success Criterion 4.1.2: Name, Role, Value ->
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html#techniques
